tests/e2e/tests/pilot/pilot_test.go

```diff
@@ -27,31 +27,28 @@ import (
 	"testing"
 	"time"
 
-	"go.uber.org/multierr"
-
 	"istio.io/istio/pilot/pkg/kube/inject"
 	"istio.io/istio/pkg/log"
 	"istio.io/istio/tests/e2e/framework"
 	"istio.io/istio/tests/util"
 )
 
 const (
-	defaultRetryBudget      = 50
-	retryDelay              = time.Second
-	httpOK                  = "200"
-	ingressAppName          = "ingress"
-	ingressContainerName    = "ingress"
-	defaultPropagationDelay = 10 * time.Second
+	defaultRetryBudget   = 50
+	retryDelay           = time.Second
+	httpOK               = "200"
+	ingressAppName       = "ingress"
+	ingressContainerName = "ingress"
 )
 
 var (
-	tc = &testConfig{
+	tc *framework.TestConfig
+	tf = &framework.TestFlags{
 		V1alpha1: true, //implies envoyv1
```

Comment:
Please fix the defaults - we never support both true. I think default should be alpha3 - so running without any options (from IDE) will test the new stuff.